lighthouse-cli/test/fixtures/form.html

```diff
@@ -0,0 +1,128 @@
+      <option value="NY">NY</option>
+      <option value="MD">MD</option>
+      <option value="OR">OR</option>
+      <option value="OH">OH</option>
+      <option value="IL">IL</option>
+      <option value="DC">DC</option>
+    </select><br>
+    <label for="zip_shipping">Zip:</label> <input type="text" id="zip_shipping"><br>
+    <input type="button" value="Submit">
+  </div>
+  <div id="billing" style="display: none">
+    <u>Billing address:</u><br>
+    <label for="name_billing">Name:</label> <input type="text" id="name_billing"><br>
+    <label for="address_billing">Address:</label> <input type="text" id="address_billing" autocomplete="billing street-address"><br>
+    <label for="city_billing">City:</label> <input type="text" id="city_billing"><br>
+    <label for="state_billing">State:</label> <select id="state_billing">
+      <option value="" selected="">Select a state</option>
+      <option value="CA">CA</option>
+      <option value="MA">MA</option>
+      <option value="NY">NY</option>
+      <option value="MD">MD</option>
+      <option value="OR">OR</option>
+      <option value="OH">OH</option>
+      <option value="IL">IL</option>
+      <option value="DC">DC</option>
+    </select><br>
+    <label for="zip_billing">Zip:</label> <input type="text" id="zip_billing"><br>
+    <input id="submit2" type="button" value="Submit">
+  </div>
+  <div id="payment">
+    <label for="name_cc2">Name on card:</label>  <textarea type="text" id="name_cc2" autocomplete="cc-name"></textarea><br>
+    <label for="CCNo2">Credit card number:</label>: <input type="text" name="CCNo2"><br>
+    <label for="CCExpiresMonth2">Expiry Date:</label>: <select id="CCExpiresMonth2">
+      <option value="">MM</option>
+      <option value="01">01</option>
+      <option value="02">02</option>
+      <option value="03">03</option>
+      <option value="04">04</option>
+      <option value="05">05</option>
+      <option value="06">06</option>
+      <option value="07">07</option>
+      <option value="08">08</option>
+      <option value="09">09</option>
+      <option value="10">10</option>
+      <option value="11">11</option>
+      <option value="12">12</option>
+    </select> / <select id="CCExpiresYear">
+      <option value="">YY</option>
+      <option value="2019">2019</option>
+      <option value="2020">2020</option>
+      <option value="2021">2021</option>
+      <option value="2022">2022</option>
+      <option value="2023">2023</option>
+      <option value="2024">2024</option>
+      <option value="2025">2025</option>
+      <option value="2026">2026</option>
+      <option value="2027">2027</option>
+      <option value="2028">2028</option>
+      <option value="2029">2029</option>
+    </select><br>
+    <label for="cvc2">CVC:</label> <input id="cvc2" autocomplete="cc-csc"><br>
+    <br>
+    <input id="submit3" type="button" value="Submit">
+  </div>
+  <div id="thanks">
+    <br>
+    <a href="/">Go back</a>
+  </div>
+
```

Comment:
nit: remove all these empty lines? might be worth running this whole thing through a formatter :)